api/api_ethereum.go

```diff
@@ -982,8 +996,7 @@ func newEthTransactionReceipt(tx *types.Transaction, b Backend, blockHash common
 	if b.ChainConfig().IsEthTxTypeForkEnabled(new(big.Int).SetUint64(blockNumber)) {
 		// TODO-Klaytn: Klaytn is using fixed BaseFee(0) as now but
 		// if we apply dynamic BaseFee, we should add calculated BaseFee instead of using params.ZeroBaseFee.
-		baseFee := new(big.Int).SetUint64(params.ZeroBaseFee)
-		fields["effectiveGasPrice"] = hexutil.Uint64(tx.EffectiveGasPrice(baseFee).Uint64())
+		fields["effectiveGasPrice"] = hexutil.Uint64(tx.EffectiveGasPrice(header.BaseFee).Uint64())
```

Comment:
I think we need to add nil or hard fork check here.

Reply:
You can ignore my comment, because i checked that `HeaderByHash` (called to pass the header parameter to this function) will return an error if header is not existed.

Reply:
Thank you. Checked.